hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -643,6 +644,12 @@ public static PropertyBuilder withPropertyBuilder() {
     private Boolean urlEncodePartitioning;
     private HoodieTimelineTimeZone commitTimeZone;
 
+    /**
+     * Persist the configs that is written at the first time, and should not be changed.
+     * Like KeyGenerator's configs.
+     */
+    private Properties others = new Properties();
```

Comment:
So far we have never added loose configs (random key value pairs) to our tableConfig. But these could be empty if not for timestamp based key gen. Can't think of better ways. 

btw, @xushiyan @codope @yihua : Adding new properties to tableConfig does not warrant a new table version right? wanted to confirm w/ you folks.